hq-graphql.gemspec

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Gemspec/RequiredRubyVersion
```

Comment:
Is this a new cop that was added? Why does it need disabled? Rubocop is passing locally for me without disabling this cop
<img width="324" alt="Screen Shot 2020-08-10 at 4 57 39 PM" src="https://user-images.githubusercontent.com/6320023/89835591-a6193280-db2a-11ea-8877-f8acf5d251a6.png">

Reply:
Yes, it is a new cop that appeared on Rubocop `0.89.0`. I disabled it because I did not want to restrict a specific Ruby version, as github actions are not being able to install Ruby 2.6.3  see this issue [(here)](https://github.com/actions/setup-ruby/issues/8#issue-479921744)
I can try by specifying  only Ruby `2.6` and see what happens

Reply:
I just disabled new cops in our default rubocop config so this shouldn't be an issue anymore. Plesae remove.